guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -192,15 +192,31 @@ If you have [Visual Studio Code](https://code.visualstudio.com) and [Docker](htt
 
 #### Using Dev Container CLI
 
-Alternatively, with [Docker](https://www.docker.com) and [npm](https://github.com/npm/cli) installed, you can run [Dev Container CLI](https://github.com/devcontainers/cli) to utilize the [`.devcontainer`](https://github.com/rails/rails/tree/main/.devcontainer) configuration from the command line.
+With [npm](https://github.com/npm/cli) and [Docker](https://www.docker.com) or [Podman](https://podman.io/) installed, you can run [Dev Container CLI](https://github.com/devcontainers/cli) to utilize the [`.devcontainer`](https://github.com/rails/rails/tree/main/.devcontainer) configuration from the command line.
 
+##### Using Docker
 ```bash
 $ npm install -g @devcontainers/cli
 $ cd rails
 $ devcontainer up --workspace-folder .
 $ devcontainer exec --workspace-folder . bash
 ```
 
+##### Using Podman
+```bash
+$ npm install -g @devcontainers/cli
+$ cd rails
+$ podman machine init
+$ podman machine start
+$ tools/devcontainer up
```

Comment:
I think `tools/devcontainer` works without `npm`/`@devcontainers/cli` installed. Assuming that's correct, this may deserve its own heading instead of being under `Using Dev Container CLI`.

Reply:
Agreed - I'll test that and create a new heading if it's not necessary.

Reply:
It works, new heading created.